compiler/dcalc/from_scopelang.ml

```diff
@@ -660,7 +660,7 @@ let translate_rule
                   scope_let_kind = Assertion;
                 },
                 next ))
-          (Bindlib.bind_var (Var.make "_") next)
+          (Bindlib.bind_var (Var.make "assert1") next)
```

Comment:
It's just for cosmetic reasons: typically a testing scope defines a series of assertions, and with this the variables will get named `assert1`, `assert2`, etc. instead of `assert`, `assert2`, etc.